Ports/.hosted_defs.sh

```diff
@@ -5,19 +5,17 @@ SCRIPT="$(dirname "${0}")"
 export SERENITY_ARCH="${SERENITY_ARCH:-i686}"
 export SERENITY_TOOLCHAIN="${SERENITY_TOOLCHAIN:-GNU}"
 
-if [ -z "${HOST_CC:=}" ]; then
-    export HOST_CC="${CC:=cc}"
-    export HOST_CXX="${CXX:=c++}"
-    export HOST_AR="${AR:=ar}"
-    export HOST_RANLIB="${RANLIB:=ranlib}"
-    export HOST_PATH="${PATH:=}"
-    export HOST_READELF="${READELF:=readelf}"
-    export HOST_OBJCOPY="${OBJCOPY:=objcopy}"
-    export HOST_STRIP="${STRIP:=strip}"
-    export HOST_PKG_CONFIG_DIR="${PKG_CONFIG_DIR:=}"
-    export HOST_PKG_CONFIG_SYSROOT_DIR="${PKG_CONFIG_SYSROOT_DIR:=}"
-    export HOST_PKG_CONFIG_LIBDIR="${PKG_CONFIG_LIBDIR:=}"
-fi
```

Comment:
Realistically, we could also just add them to the list of exceptions.

Reply:
Is there a case in which it is run twice?

Reply:
Not intentionally as far as I can imagine, it's probably more of a safeguard.

Reply:
So should I drop this change then? Just making sure.

Reply:
Presumably yes.